scripts/parse_existing_cohort.py

```diff
@@ -139,12 +139,12 @@ async def get_read_filenames(
         """
 
         return [
```

Comment:
This will now return m5's, and cause downstream errors. Maybe just filtering to match
```suggestion
        relevant_extensions = ('.cram', '.fastq.gz', '.fastq', 'fq.gz', '.fq', '.bam')
        return [
            filename
            for filename, path in self.filename_map.items()
            if fastq_file_name_to_sample_id(filename) == row[Columns.MANIFEST_FLUID_X]
            and any(filename.endswith(ext) for ext in relevant_extensions)
        ]
        
```

Reply:
I don't know too much about this parser, so happy to just accept this suggestion :)